validator/db/kv/attestation_history.go

```diff
@@ -35,6 +37,21 @@ func (store *Store) AttestationHistoryForPubKeys(
 			} else {
 				attestationHistory = enc
 			}
+			var minAtt attHist.MinAttestation
+			var setMin bool
+			enc = bucket.Get(attHist.GetMinSourceKey(key))
+			if len(enc) != 0 {
+				minAtt.Source = bytesutil.BytesToUint64BigEndian(enc)
+				setMin = true
+			}
+			enc = bucket.Get(attHist.GetMinTargetKey(key))
```

Comment:
Does this mean that our attestinghistory package knows about boltdb keys? Is there a way we can better separate those concerns so they are independent?